app/services/analytics_events.rb

```diff
@@ -606,6 +615,110 @@ def idv_address_visit
+  # @param [Boolean] acuant_sdk_upgrade_a_b_testing_enabled
+  # @param [String] acuant_version
+  # @param [Boolean] assessment
+  # @param [Integer] attempt number of attempts
+  # @param [String] documentType
+  # @param [Integer] dpi  dots per inch of image
+  # @param [Integer] failedImageResubmission
+  # @param [String] fingerprint fingerprint of the image added
+  # @param [String] flow_path whether the user is in the hybrid or standard flow
+  # @param [Integer] glare
+  # @param [Integer] glareScoreThreshold
+  # @param [Integer] height height of image added in pixels
+  # @param [Boolean] isAssessedAsBlurry
+  # @param [Boolean] isAssessedAsGlare
+  # @param [Boolean] isAssessedAsUnsupported
+  # @param [String] mimeType MIME type of image added
+  # @param [Integer] moire
+  # @param [String] phone_question_ab_test_bucket
+  # @param [Integer] sharpness
+  # @param [Integer] sharpnessScoreThreshold
+  # @param [Integer] size size of image added in bytes
+  # @param [String] source
+  # @param [Boolean] use_alternate_sdk
+  # @param [Integer] width width of image added in pixels
+  # Back image was added in document capture
+  # rubocop:disable Naming/VariableName,Naming/MethodParameterName
+  def idv_back_image_added(
+    acuantCaptureMode:,
+    acuant_sdk_upgrade_a_b_testing_enabled:,
+    acuant_version:,
+    assessment:,
+    attempt:,
+    documentType:,
+    dpi:,
+    failedImageResubmission:,
+    fingerprint:,
+    flow_path:,
+    glare:,
+    glareScoreThreshold:,
+    height:,
+    isAssessedAsBlurry:,
+    isAssessedAsGlare:,
+    isAssessedAsUnsupported:,
+    mimeType:,
+    moire:,
+    phone_question_ab_test_bucket:,
+    sharpness:,
+    sharpnessScoreThreshold:,
+    size:,
+    source:,
+    use_alternate_sdk:,
+    width:,
+    **_extra
+  )
+    track_event(
+      'Frontend: IdV: back image added', # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
+      acuantCaptureMode: acuantCaptureMode,
+      acuant_sdk_upgrade_a_b_testing_enabled: acuant_sdk_upgrade_a_b_testing_enabled,
+      acuant_version: acuant_version,
+      assessment: assessment,
+      attempt: attempt,
+      documentType: documentType,
+      dpi: dpi,
+      failedImageResubmission: failedImageResubmission,
+      fingerprint: fingerprint,
+      flow_path: flow_path,
+      glare: glare,
+      glareScoreThreshold: glareScoreThreshold,
+      height: height,
+      isAssessedAsBlurry: isAssessedAsBlurry,
+      isAssessedAsGlare: isAssessedAsGlare,
+      isAssessedAsUnsupported: isAssessedAsUnsupported,
+      mimeType: mimeType,
+      moire: moire,
+      phone_question_ab_test_bucket: phone_question_ab_test_bucket,
+      sharpness: sharpness,
+      sharpnessScoreThreshold: sharpnessScoreThreshold,
+      size: size,
+      source: source,
+      use_alternate_sdk: use_alternate_sdk,
+      width: width,
+    )
+  end
+  # rubocop:enable Naming/VariableName,Naming/MethodParameterName
+
+  def idv_back_image_clicked(**_extra)
```

Comment:
To the earlier point about this `**_extra`, can we just exclude it? Or is the problem that the `send` will always try to pass a hash of values; in which case, could we make sure that only happens if it's non-empty? I guess I'm thinking that we should either never expect `**extra` to be passed, or we could just log the `**extra` for consistency with other events (except for methods like this which don't have any arguments).

Reply:
We're limited by https://github.com/18F/identity-idp/blob/d099da2181ab73c733f8a0dea561ca4152e5aa10/lib/analytics_events_documenter.rb#L114-L116 which ends up failing lint with:

```
app/services/analytics_events.rb:704 idv_back_image_clicked missing **extra
```